internal/strict/controls/controls.go

```diff
@@ -38,9 +38,6 @@ const (
 	// SkipDependenciesInputs is the control that prevents reading dependencies inputs and get performance boost.
 	SkipDependenciesInputs = "skip-dependencies-inputs"
 
-	// RequireExplicitBootstrap is the control that prevents the backend for remote state from being bootstrapped unless the `--backend-bootstrap` flag is specified.
-	RequireExplicitBootstrap = "require-explicit-bootstrap"
```

Comment:
We shouldn't remove strict controls, we should only complete them. That prevents the breaking change of users using `--strict-control require-explicit-bootstrap`. They'll just get a warning instead this way.